src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/ResourceQueriesQueryString.feature.disabled

```diff
@@ -0,0 +1,58 @@
+Feature: Query String handling for GET requests for Resource Queries
+
+        Background:
+            Given a school exists
+                  | schoolId  | nameOfInstitution      | gradeLevels       | educationOrganizationCategories |
+                  | 255901001 | Grand Bend High School | [ "Tenth grade" ] | [ "School" ]                    |
+              And there is a valid AcademicWeek
+                  | weekIdentifier | beginDate  | endDate    | totalInstructionalDays |
+                  | Week One       | 2024-05-15 | 2024-05-15 | 2                      |
+
+        @ignore
+        Scenario: Ensure clients can GET information when querying by valid date
+             When sending a GET request to "/ed-fi/academicWeeks?beginDate={beginDate}"
```

Comment:
I think is better if we replace the {beginDate} with the current value used in the Background section since this already exists

Reply:
Complete